src/03_modules.py

```diff
@@ -10,22 +10,23 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(sys.argv)
```

Comment:
Make sure you take the time to really understand "sys.argv"! It will serve you well in the future! 👍 
![Screen Shot 2019-12-09 at 10 02 32 AM](https://user-images.githubusercontent.com/16110931/70446492-2f8fd680-1a6b-11ea-915e-c3350f30ef66.png)